### PR TITLE
feat(web-search): add support for Brave search engine

### PR DESCRIPTION
**Pull request by Bryan-netizen**
Added brave browser to the web search list, as well as my own alias.

## Standards checklist:

- [ ] The PR title is descriptive.
- [ ] The PR doesn't replicate another PR which is already open.
- [ ] I have read the contribution guide and followed all the instructions.
- [ ] The code follows the code style guide detailed in the wiki.
- [ ] The code is mine or it's from somewhere with an MIT-compatible license.
- [ ] The code is efficient, to the best of my ability, and does not waste computer resources.
- [ ] The code is stable and I have tested it myself, to the best of my abilities.
- [ ] If the code introduces new aliases, I provide a valid use case for all plugin users down below.

## Changes:

- [...]

## Other comments:

...

### CODE REVIEW

File: plugins/web-search/web-search.plugin.zsh [hunk 1]
```diff
@@ -47,6 +48,7 @@ function web_search() {
 
 
 alias bing='web_search bing'
+alias br='web_search brave'
```
Comment (carlosala) [inline plugins/web-search/web-search.plugin.zsh:51]: Could you change that to `brave`? `br` might exist in people's path, could be a tipical alias for whatever!

**Review by carlosala — CHANGES_REQUESTED** [review]
I like it!

File: plugins/web-search/web-search.plugin.zsh (hunk 1 above)
Comment (mcornella) [reply plugins/web-search/web-search.plugin.zsh:51]: `brave` might be the command to start the browser. An alternative might be `brave_search`, which is not likely to override anything, but I'm not satisfied with it either.

Comment (Bryan-netizen) [reply plugins/web-search/web-search.plugin.zsh]: Made the changes, thank you for the feedback.

**Review by larson-carter — COMMENTED** [review]
I like this a lot as well. Good by me.

**Comment by mcornella** [comment]
The `web_search` parameter can be `brave`, my point was that the alias could not be named the same way.

`br` is already a command: https://command-not-found.com/br

My advice is to name the alias something longer, like `braves` or something else like `brave` + an abbreviation of search.

**Comment by Bryan-netizen** [comment]
Thanks for helping me with my 1st contribution, I really appreciate it!

**Pull request merged**